apps/app/src/app/services/app.service.ts

```diff
@@ -40,12 +40,30 @@ export class AppService {
 
   private listenLanguageChange(): void {
     this.translateService.onLangChange.subscribe((event: LangChangeEvent) => {
-      this.currentLocaleSubject.next(this.getLocaleFromLang(event.lang))
+      this.currentLocaleSubject.next(this.getLocaleFromLang(event.lang));
     });
   }
 
   private getLocaleFromLang(lang: string) {
     return this.config.getLocales().includes(lang) ? lang : 'en-US';
   }
 
+  checkEmptyKBAlert() {
```

Comment:
It took me a while to understand why you were mapping the counter to a boolean, so maybe renaming (and typing) this method would be better. 
I'm not sure what would be a good name for it though. `hasResources` ? `isEmptyKb`? `isKbStillEmptyAfterFirstDay`?